### ISSUE TITLE
FramePack training loss

### ISSUE THREAD

**Issue by wlike** [kohya-ss/musubi-tuner#315]
Does anyone use this repo to train a FramePack lora for the i2v task successfully?
What does the loss curve look like? And what are the hyperparameters used?

**Comment by xianienie** [comment]
This is my training parameter
/root/autodl-tmp/musubi-tuner-main/fpack_train_network.py --mixed_precision bf16 --dynamo_backend INDUCTOR --dynamo_mode max-autotune --fp8_scaled --fp8_base --image_encoder ./ckpts/framepack/sigclip_vision_patch14_384.safetensors --dit ./ckpts/framepack/FramePackI2V_HY_bf16.safetensors --vae ./ckpts/framepack/hunyuan_video_vae_fp32.safetensors --text_encoder1 ./ckpts/text_encoder/llava_llama3_fp16.safetensors --text_encoder2 ./ckpts/text_encoder_2/clip_l.safetensors --latent_window_size 9 --dataset_config ./toml/qinglong-video-datasets.toml --split_attn --flash_attn --logging_dir ./logs --optimizer_type=pytorch_optimizer.ADOPT --optimizer_args cautious=True --learning_rate 3.5e-4 --lr_scheduler cosine_with_min_lr --lr_warmup_steps 0 --lr_decay_steps 0.2 --lr_scheduler_num_cycles 1 --lr_scheduler_power 1 --lr_scheduler_timescale 0 --lr_scheduler_min_lr_ratio 0.3 --blocks_to_swap 4 --vae_spatial_tile_sample_min_size 256 --vae_chunk_size 32 --img_in_txt_in_offloading --log_with wandb --wandb_run_name pubg_146_all_dim64_shift6_sig16 --wandb_api_key e2e22b47a5c1f225f8fc618e04615544e4787464 --log_config --gradient_checkpointing --max_data_loader_n_workers 8 --persistent_data_loader_workers --network_module networks.lora_framepack --network_dim 64 --network_alpha 32 --network_dropout 0 --network_args loraplus_lr_ratio=4 --timestep_sampling shift --discrete_flow_shift 6 --sigmoid_scale 1.6 --max_train_epochs 60 --save_every_n_epochs 1 --save_last_n_epochs 4 --seed 42113 --guidance_scale 1 --gradient_accumulation_steps 1 --output_dir ./output --output_name pubg_146_all_dim64_shift6_sig16 --max_grad_norm 1

![Image](https://github.com/user-attachments/assets/28647b8f-fca4-4f17-8497-1d12bf27f929)

This is the loss value I obtained after training with 14 videos, each ranging from 150 to 200 frames in length, repeated twice, over 60 epochs.

In the training test of a video, I discovered that a high discrete_flow_shift value >3 results in a higher fidelity restoration of the source data. However, it prevents the migration of actions. When using other images with identical actions for inference, the generated videos tend to be static, I attempted to reduce the LoRA weight to approximately 0.6 during the inference process, which successfully enhanced the dynamic quality of the transferred video. However, the resulting actions were not similar to those in the original video and lacked logical coherence. discrete_flow_shift value higher also increases the initial loss during training. To address this, I attempted to smooth the processing by increasing the sigmoid_scale, but I have yet to find the optimal value.
I haven’t found an effective solution to address the issue of generalization in action transfer yet.

**Comment by yahya280101** [comment]
have you found anything for action transfer @xianienie , trying some data that relies heavily on motion correctness.
